Dshop/apps/products_catalogue/views.py

```diff
@@ -112,3 +112,19 @@ def get(self, request, *args, **kwargs):
 
 class CeneoAPIException(Exception):
     pass
+
+
+class CategoryListView(ListView):
+    model = Category
+    context_object_name = 'categories'
+    template_name = 'products_catalogue/categories_list.html'
+
+    def get_queryset(self):
+        return Category.objects.filter(is_active=True)
```

Comment:
```suggestion
        return self.model.objects.filter(is_active=True)
```
Try this. This way it uses model defined earlier.